.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,125 @@
+---
+name: Release
+about: Cut a Crossplane release
+labels: release
+---
+
+<!--
+Issue title should be in the following format:
+
+    Cut vX.Y.Z Release on DATE
+
+For example:
+
+    Cut v1.3.0 on June 29, 2021.
+
+Please assign the release manager to the issue.
+-->
+
+### Steps
+<!--
+Please complete the following steps in order. Links should be populated at the
+bottom of this section.
+-->
+
+This issue can be closed when we have:
+
+<!--
+Please uncomment the following block only if cutting a minor release. Most of
+these should be completed at the beginning of Code Freeze:
+https://crossplane.io/docs/v1.2/reference/release-cycle.html#code-freeze
+
+The exception is the Crossplane docs website repo update. You can open a PR at
+code freeze time, but it should not be merged until the release is complete.
+-->
+
+<!--
+- [ ] Created the [release branch][release-branch].
+- [ ] Created and merged an empty commit to the `master` branch ([PR][empty-commit-pr]).
+- [ ] Run the [Tag workflow][rc-tag-workflow] on the `master` branch with the next release candidate tag ([Tag][rc-tag]).
+- [ ] Updated the current release version in the [Crossplane docs website repo] ([link][website-pr]).
+- [ ] Updated the release branch reaching EOL with docs removal directive ([link][eol-pr]).
+-->
+- [ ] Updated all version information in the documentation on the relevant release branch ([link][docs-update-pr]).
```

Comment:
Seems to list tasks for a Crossplane rather than provider release.

Reply:
This should be updated to the provider-template specific workflow.

Reply:
I added this template unintentionally - I don't think we should include it at all. Most providers don't currently have such an issue template.